Common/FeatureFlags.swift

```diff
@@ -139,7 +155,8 @@ extension FeatureFlagConfiguration : CustomDebugStringConvertible {
             "* scenariosEnabled: \(scenariosEnabled)",
             "* sensitivityOverridesEnabled: \(sensitivityOverridesEnabled)",
             "* simulatedCoreDataEnabled: \(simulatedCoreDataEnabled)",
-            "* walshInsulinModelEnabled: \(walshInsulinModelEnabled)"
+            "* siriEnabled: \(siriEnabled)",
+            "* automaticBolusEnabled: \(automaticBolusEnabled)"
```

Comment:
add `outsideDosesEnabled`:
```suggestion
            "* automaticBolusEnabled: \(automaticBolusEnabled)",
            "* outsideDosesEnabled: \(outsideDosesEnabled)",
```